appium/webdriver/webdriver.py

```diff
@@ -176,6 +284,24 @@ def __init__(
         By.IMAGE = MobileBy.IMAGE
         By.CUSTOM = MobileBy.CUSTOM
 
+        self._extensions = extensions
+        for extension in self._extensions:
+            instance = extension(self.execute)
```

Comment:
we use `instance.method_name()` a lot. should we just make it a variable for readability?

```
name = instance.method_name()
```
or
```
method_name = instance.method_name()
```